content/id/docs/concepts/services-networking/dual-stack.md

```diff
@@ -0,0 +1,145 @@
+      * `--node-cidr-mask-size-ipv4|--node-cidr-mask-size-ipv6` _default_ /24 
+        untuk IPv4 dan /64 untuk IPv6
+   * kubelet:
+      * `--feature-gates="IPv6DualStack=true"`
+   * kube-proxy:
+      * `--proxy-mode=ipvs`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
+      * `--feature-gates="IPv6DualStack=true"`
+
+{{< caution >}}
+Jika kamu menentukan blok alamat IPv6 yang lebih besar dari /24 melalui 
+`--cluster-cidr` pada baris perintah, maka pengalokasian tersebut akan gagal.
+{{< /caution >}}
+
+## Layanan-layanan
+
+Jika klaster kamu mengaktifkan jaringan _dual-stack_ IPv4/IPv6, maka kamu dapat 
+membuat {{<glossary_tooltip text ="Service" term_id ="service">}} (layanan) dengan 
+alamat IPv4 atau IPv6. Kamu dapat memilih keluarga alamat untuk IP klaster  
+layanan kamu dengan mengatur bagian, `.spec.ipFamily`, pada layanan tersebut.
+Kamu hanya dapat mengatur bagian ini saat membuat layanan baru. Mengatur bagian 
+`.spec.ipFamily` bersifat opsional dan hanya boleh digunakan jika kamu berencana
+untuk mengaktifkan {{<glossary_tooltip text ="Service "term_id ="service">}} 
+IPv4 dan IPv6 dan {{<glossary_tooltip text ="Ingress "term_id ="ingress">}}
+pada klaster kamu. Konfigurasi bagian ini bukan persyaratan untuk lalu lintas 
+[_egress_] (#trafik-egress).
+
+{{< note >}}
+Keluarga alamat _default_ untuk klaster kamu adalah keluarga alamat dari rentang 
+IP cluster layanan pertama yang dikonfigurasi melalui flag 
+`--service-cluster-ip-range` pada kube-controller-manager.
+{{< /note >}}
+
+Kamu dapat mengatur `.spec.ipFamily` ke salah satu dari:
+
+   * `IPv4`: Dimana server API akan mengalokasikan IP dari `service-cluster-ip-range` yaitu `ipv4`
+   * `IPv6`: Dimana server API akan mengalokasikan IP dari `service-cluster-ip-range` yaitu `ipv6`
+
+Spesifikasi layanan berikut ini tidak memasukkan bagian `ipFamily`.
+Sehingga Kubernetes akan mengalokasikan alamat IP (atau yang dikenal juga sebagai 
+"cluster IP") dari `service-cluster-ip-range` yang dikonfigurasi pertama kali
+untuk Service ini.
+
+{{< codenew file="service/networking/dual-stack-default-svc.yaml" >}}
+
+Spesifikasi layanan berikut memasukkan bagian `ipFamily`. Sehingga Kubernetes
+akan mengalokasikan alamat IPv6 (atau yang dikenal juga sebagai "cluster IP") 
+dari `service-cluster-ip-range` yang dikonfigurasi untuk layanan ini.
+
+{{< codenew file="service/networking/dual-stack-ipv6-svc.yaml" >}}
+
+Sebagai perbandingan, spesifikasi layanan berikut ini akan dialokasikan alamat
+IPv4 (atau yang dikenal juga sebagai "cluster IP") dari `service-cluster-ip-range` 
+yang dikonfigurasi untuk layanan ini.
+
+{{< codenew file="service/networking/dual-stack-ipv4-svc.yaml" >}}
+
+### Tipe _LoadBalancer_
+
+Penyedia layanan _cloud_ yang mendukung IPv6 untuk pengaturan beban eksternal,
+Pengaturan bagian `type` ke` LoadBalancer` sebagai tambahan dalam mengatur bagian 
+`ipFamily` ke `IPv6` menyediakan sebuah _cloud load balancer_ untuk layanan kamu.
+
+## Trafik _egress_
```

Comment:
```suggestion
## Lalu Lintas _egress_
```
Trafik is not found in KBBI, wdyt if we use Lalu Lintas?

Reply:
Ok!